geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -13,6 +13,25 @@
     {# translators: label for checkbox toggle to show the translation panel for a document #}
     <input type="checkbox" id="translation-on" aria-label="{% translate 'show translation' %}" disabled="true"/>
     <label for="translation-on"><svg><use xlink:href="{% static 'img/ui/all/all/translation-toggle.svg' %}#translation-toggle" /></svg></label>
+    <div class="dropdown-container">
+        <div></div>
```

Comment:
are the empty divs for matching the panel sizes?

Reply:
Yes!